test-cases/longevity/longevity-lwt-500G-3d.yaml

```diff
@@ -1,10 +1,9 @@
-# Test duration 25 hours temporary. Should be 3 days. Will be changed later
-test_duration: 1500
+test_duration: 4350
 prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml n=400000000 ops'(insert_query=1)' cl=QUORUM -mode native cql3 -rate threads=1000" ]
-stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1,lwt_deletes=1)' cl=QUORUM duration=900m -mode native cql3 -rate threads=10" ,
-             "cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1,lwt_deletes=1)' cl=QUORUM duration=900m -mode native cql3 -rate threads=10"
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1,lwt_deletes=1)' cl=QUORUM duration=2000m -mode native cql3 -rate threads=10" ,
```

Comment:
if the test should run 3 days (4320 minutes) why do you set the command to run 2000m?
does "prepare_write" command take 2320 minutes?